arch/arm/src/nrf52/nrf52_ppi.c

```diff
@@ -0,0 +1,134 @@
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+
+#include "arm_arch.h"
+#include "chip.h"
+#include "nrf52_ppi.h"
+#include "hardware/nrf52_ppi.h"
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Private Types
+ ****************************************************************************/
+
+/****************************************************************************
+ * Private Function Prototypes
+ ****************************************************************************/
+
+/****************************************************************************
+ * Private Data
+ ****************************************************************************/
+
+/****************************************************************************
+ * Public Data
+ ****************************************************************************/
+
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: nrf52_ppi_channel_enable
+ ****************************************************************************/
+
+void nrf52_ppi_channel_enable(uint8_t ch, bool enable)
+{
+  DEBUGASSERT(ch < NRF52_PPI_NUM_CHANNELS);
+
+  if (enable)
+    {
+      putreg32(PPI_CHEN_CH(ch), NRF52_PPI_CHENSET);
+    }
+  else
+    {
+      putreg32(PPI_CHEN_CH(ch), NRF52_PPI_CHENCLR);
+    }
+}
+
+/****************************************************************************
+ * Name: nrf52_ppi_set_event_ep
+ ****************************************************************************/
+
+void nrf52_ppi_set_event_ep(uint8_t ch, uint32_t event_reg)
+{
+  DEBUGASSERT(ch < NRF52_PPI_NUM_CONFIGURABLE_CHANNELS);
+
+  putreg32((uint32_t)event_reg, NRF52_PPI_CHEEP(ch));
```

Comment:
```suggestion
  putreg32(event_reg, NRF52_PPI_CHEEP(ch));
```

Reply:
thanks, missed that. just force pushed to include this in last commit